### PR TITLE
add check for python3 str serializing #33

### PR DESCRIPTION
**Pull request by nyxaria**
https://github.com/uos/rospy_message_converter/issues/33

### CODE REVIEW

File: src/rospy_message_converter/message_converter.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def _convert_to_ros_time(field_type, field_value):
     return time
 
 def _convert_to_ros_primitive(field_type, field_value):
-    if field_type == "string":
+    if field_type == "string" and not python3:
```
Comment (mintar) [inline src/rospy_message_converter/message_converter.py:149]: Perhaps add a small comment here, something along the lines of:

```suggestion
    # std_msgs/msg/_String.py already calls encode() on python3, so don't do it here
    if field_type == "string" and not python3:
```

**Comment by nyxaria** [comment]
committed the comment and squashed. I think it's ready for a merge

**Comment by nyxaria** [comment]
@mintar should we merge?

**Comment by mintar** [comment]
Merged. Thanks!

**Pull request merged**